test/iron-overlay-behavior.html

```diff
@@ -254,7 +254,8 @@
         test('backdrop is removed when the element is removed from DOM', function(done) {
           runAfterOpen(overlays[0], function() {
             var backdrop = overlays[0].backdropElement;
-            overlays[0].remove();
+            Polymer.dom(backdrop.parentNode).removeChild(backdrop);
+            Polymer.dom.flush();
```

Comment:
I think Polymer.dom.removeChild (and other DOM operations) is sometimes lazy, and flush causes it to be sync.